.cirrus.yml

```diff
@@ -259,13 +259,13 @@ task:
     FILE_ENV: "./ci/test/00_setup_env_i686_multiprocess.sh"
 
 task:
-  name: '[no wallet] [bionic]'
+  name: '[no wallet, libbitcoinkernel] [bionic]'
```

Comment:
in the ci commit: pls remove the @MarcoFalke

Reply:
I removed the "@" from the commit message, is that sufficient?